.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules
 .vscode
 .idea
 *.log
+index.html
```

Comment:
You don't want to do this. You might have accidentally clicked "add to gitignore". Open this file and delete this line, then save and commit again.